### ISSUE TITLE
Introduce the dropout technique to prevent neural networks from overfitting 

### ISSUE THREAD

**Issue by beomyeol** [snuspl/cruise#583]
The dropout technique described in [Dropout: A Simple Way to Prevent Neural Networks from Overfitting](http://jmlr.org/papers/v15/srivastava14a.html) is known as an effective method to prevent neural networks from overfitting. It is widely used in many neural network models.
We should add the dropout to improve neural network performance.
